infrastructure/code/template/details.html

```diff
@@ -732,81 +757,74 @@ <h2 class="severity-title">{{.IssueTitle}}</h2>
-                      <p class="sn-loading-description">Examining the vulnerability code without having to execute the program.</p>
+                      <p class="sn-loading-description">Examining the vulnerability code without having to execute the
+                        program.</p>
                     </div>
                     <div class="sn-loading-message sn-msg-4">
                       <span class="sn-loading-title">4<span class="font-light">/4</span> Inferencing...</span>
-                      <p class="sn-loading-description">Feeding the reduced code to our neural network to obtain the prediction.</p>
+                      <p class="sn-loading-description">Feeding the reduced code to our neural network to obtain the
+                        prediction.</p>
                     </div>
                   </div>
                 </div>
               </div>
             </section>
 
             <!-- AI fixes -->
-            <section id="fixes-section" class="sn-ai-fixes hidden">
+            <section id="fixes-section" class="sn-ai-fixes delimiter-top hidden">
               <div id="info-no-diffs" class="font-light">
                 There are no fix diffs for this issue.
               </div>
               <div id="diff-top" class="row between">
-                <div id="info-top" class="font-light">Here are <span id="diff-number"></span> AI-generated solutions:</div>
+                <div id="info-top" class="font-light">Here are <span id="diff-number"></span> AI-generated solutions:
+                </div>
                 <div class="diffs-nav">
                   <span id="previous-diff" class="arrow" title="Previous diff">
-                    <svg class="arrow-icon dark-only" width="10" height="12" viewBox="0 0 10 12
-  " fill="none" xmlns="http://www.w3.org/2000/svg">
-  <path d="M8.86723 11.4303L8.86721 11.4302L0.641823 6.22447L0.387031 6.62706L0.641821 6.22447C0.532336 6.15518 0.5 6.06763 0.5 6.00001C0.5 5.93239 0.532336 5.84484 0.641821 5.77555L0.641824 5.77555L8.86721 0.569741L8.86723 0.569731C9.00417 0.483055 9.17298 0.480315 9.31053 0.543871C9.44734 0.607082 9.5 0.705333 9.5 0.79421V11.2058C9.5 11.2947 9.44734 11.3929 9.31054 11.4561C9.173 11.5197 9.00418 11.5169 8.86723 11.4303Z" stroke="#CCCCCC"/>
-  </svg>
-                    <svg class="arrow-icon light-only" width="10" height="12" viewBox="0 0 10 12
-  " fill="none" xmlns="http://www.w3.org/2000/svg">
-  <path d="M8.86723 11.4303L8.86721 11.4302L0.641823 6.22447L0.387031 6.62706L0.641821 6.22447C0.532336 6.15518 0.5 6.06763 0.5 6.00001C0.5 5.93239 0.532336 5.84484 0.641821 5.77555L0.641824 5.77555L8.86721 0.569741L8.86723 0.569731C9.00417 0.483055 9.17298 0.480315 9.31053 0.543871C9.44734 0.607082 9.5 0.705333 9.5 0.79421V11.2058C9.5 11.2947 9.44734 11.3929 9.31054 11.4561C9.173 11.5197 9.00418 11.5169 8.86723 11.4303Z" stroke="#3B3B3B"/>
-  </svg>
+                    {{.ArrowLeftDark}}
+                    {{.ArrowLeftLight}}
```

Comment:
Still need to work on showing the arrow based on the theme.

Reply:
I think you could do this by applying styles to the svg, since it looks like the only difference between them is the colour stroke

Reply:
You are right! This seems the change we need: https://github.com/snyk/vscode-extension/pull/472